friendsofsymfony/oauth-server-bundle/1.6/src/Repository/AccessTokenRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\AccessToken;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
+ * @method AccessToken|null find($id, $lockMode = null, $lockVersion = null)
+ * @method AccessToken|null findOneBy(array $criteria, array $orderBy = null)
+ * @method AccessToken[]    findAll()
+ * @method AccessToken[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class AccessTokenRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, AccessToken::class);
+    }
+
+//    /**
+//     * @return AccessToken[] Returns an array of AccessToken objects
+//     */
+    /*
+    public function findByExampleField($value)
```

Comment:
I think this code in comments should be removed. 

Add a link to the docs instead.